taxonomies/categories.txt

```diff
@@ -35894,11 +35914,16 @@ nl:Opgemaakte salades
 
 <en:Prepared salads
 <en:Meals with fish
-en:Tuna salad
+en:Tuna salad, Prepared mixed meat/fish canned drained salad
 de:Thunfischsalat
 fi:tonnikalasalaatti
-fr:Salades au thon
 nl:Tonijnsalades
+fr:Salades au thon, salade composée avec viande apertisée égouttée, salade composée avec poisson appertisée égouttée
```

Comment:
If a CIQUAL category matches 2 or more different categories, then we need to apply the CIQUAL category to all those different OFF categories, but we should not make it one category on OFF. So we do want to apply this ciqual category to "tuna salads", but we don't want to make those 2 new entries synonyms of it.